src/main/java/optimus/Ui.java

```diff
@@ -0,0 +1,96 @@
+package optimus;
+
+import java.util.ArrayList;
+
+import javafx.application.Platform;
+
+// Let ChatGPT check and suggest comments and JavaDocs according to CS2103T style guide
+/**
+ * Handles interactions with the user including displaying messages and reading input.
+ */
+public class Ui {
+    private MainWindow mainWindow;
+
+    public Ui(MainWindow mainWindow) {
+        this.mainWindow = mainWindow;
+    }
+
+    // Method to show messages to the user in the chatbox
```

Comment:
should the first sentence start in the form Returns ..., Sends ..., Adds ... etc. (not Return or Returning etc.)